iwyu_preprocessor.cc

```diff
@@ -1087,4 +1129,25 @@ bool IwyuPreprocessorInfo::ForwardDeclareIsInhibited(
       ContainsKey(*inhibited_forward_declares, normalized_symbol_name);
 }
 
+bool IwyuPreprocessorInfo::ForwardDeclareInKeepRange(
+    const SourceLocation& loc) const {
```

Comment:
`SourceLocation` is a value type, so you can pass it by val instead. That also makes the definition fit on one 80-column line.